dev-infra/release/publish/actions.ts

```diff
@@ -494,9 +496,10 @@ export abstract class ReleaseAction {
    * @param newVersion The new version to be published.
    * @param publishBranch Name of the branch that contains the new version.
    * @param npmDistTag NPM dist tag where the version should be published to.
+   * @param prerelease Whether the release is a prerelease, i.e. a next version.
```

Comment:
Aren't RCs prereleases too?

```suggestion
   * @param prerelease Whether the release is a prerelease, i.e. a next or RC version.
```

Or did you mean "a version tagged as 'next' on NPM" (in which case, why not use `npmDistTag` to compute `prerelease` :grin:)?